crates/ere-pico/Cargo.toml

```diff
@@ -8,8 +8,11 @@ license.workspace = true
 [dependencies]
 zkvm-interface = { workspace = true }
 thiserror = "2"
-pico-sdk = { git = "https://github.com/brevis-network/pico", tag = "v1.1.3" }
+pico-sdk = { git = "https://github.com/brevis-network/pico", tag = "v1.1.4" }
```

Comment:
Tried this to fix Pico but looks like is still broken.

Reply:
We can not build pico and create an issue -- it's not really possible to test it right now due to the memory overhead of CPU proving

Reply:
https://github.com/eth-act/ere/issues/49

Reply:
Disabled it, so now looks like this https://github.com/eth-act/ere/actions/runs/16168531338
The workflow file has a reference to #49 to explain why.